### PR TITLE
EPMRPP-38628 REACT. Overall Statistic. Sub-defect type isn't shown in the legend o…

### PR DESCRIPTION
**Pull request by vitalii-chornyi**
…f the widget

### CODE REVIEW

File: app/src/components/widgets/charts/launchExecutionAndIssueStatistics/issueStatisticsChart.jsx [hunk 1]
```diff
@@ -152,40 +152,80 @@ export class IssueStatisticsChart extends Component {
...
+    const { defectTypes } = this.props;
+
+    return columns.reduce((colors, column) => {
+      const locator = getItemNameConfig(column[0]).locator;
+      const defectTypesValues = Object.values(defectTypes);
+
+      for (let i = 0; i < defectTypesValues.length; i += 1) {
+        const defect = defectTypesValues[i].find((defectType) => defectType.locator === locator);
+
+        if (defect) {
```
Comment (ultd39) [inline app/src/components/widgets/charts/launchExecutionAndIssueStatistics/issueStatisticsChart.jsx]:
```
return defect ? {
    ...colors,
    [column[0]]: defect.color 
} : colors;
```

Comment (vitalii-chornyi) [reply app/src/components/widgets/charts/launchExecutionAndIssueStatistics/issueStatisticsChart.jsx]: No, we can't break the loop on first iteration, we need to find first match and then we can break

Comment (ultd39) [reply app/src/components/widgets/charts/launchExecutionAndIssueStatistics/issueStatisticsChart.jsx]: Oh, you right! Sorry.

**Comment by DiscoElevator** [comment]
Is there an issue number?

**Comment by vitalii-chornyi** [comment]
It's shown in the branch name but I committed w/o issue number somehow :)

**Pull request merged**